<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminEditCatalogue</class>
 <widget class="QMainWindow" name="adminEditCatalogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenuBar {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 lightgray, stop:1 darkgray);
    spacing: 3px; /* spacing between menu bar items */
}

QMenuBar::item {
    padding: 1px 4px;
    background: transparent;
    border-radius: 4px;
}

QMenuBar::item:selected { /* when selected using mouse or keyboard */
    background: #a8a8a8;
}

QMenuBar::item:pressed {
    background: #888888;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
	background: #FFE5D0;
	font-family: Montserrat;
}

QLineEdit {
	border: 1px solid;
	border-radius: 15px;
	border-color: #e5e5e5;
	background-color: palette(base);
	padding: 0px 5px;
	font-size: 13px;
}

QPushButton {
	border-radius: 15px; 
	font-size: 13px;
}</string>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>21</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Line{
	color:rgb(221, 46, 68);
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>931</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	border-radius: 25px;
	background: rgb(255, 255, 255);
}

QLabel{
	background: rgb(255, 255, 255);
	font-size: 14px;
	font-weight: bold;

}

QLineEdit {
	background:rgb(255, 229, 208);
	border: 1px solid;
	border-radius: 15px;
	border-color: #e5e5e5;
	background-color: palette(base);
	padding: 0px 5px;
}

QTextEdit {
	background:rgb(255, 229, 208);
	border: 1px solid;
	border-radius: 15px;
	border-color: #e5e5e5;
	background-color: palette(base);
	padding: 0px 5px;
}

QComboBox{
	background:rgb(255, 229, 208);
	border: 1px solid;
	border-radius: 15px;
	border-color: #e5e5e5;
	background-color: palette(base);
	padding: 0px 5px;
}

QFormLayout{
	color: rgb(255, 255, 255);
}


QPushButton {
	border-radius: 15px; 
	font-size: 13px;
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="confirmBookDetails">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>350</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: #E78A6B;
color:  #fff;
font-weight: 500;
}

QPushButton:hover {
background-color: rgba(231, 138, 107, 0.8);
}

QPushButton:pressed {
background-color: rgb(196, 116, 90);
}</string>
     </property>
     <property name="text">
      <string>Add Book</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	font-weight: bold;
	font-size: 18px;

}</string>
     </property>
     <property name="text">
      <string>Book Details</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_39">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>282</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777185</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Enter book 
 description</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>97</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter Genre</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bookGenre">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>97</y>
       <width>601</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>134</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Amount of Copies</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bookCopies">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>134</y>
       <width>601</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>171</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Select Image</string>
     </property>
    </widget>
    <widget class="QComboBox" name="selectedImage_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>171</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
border-radius: 15px;
background-color: palette(base);
font-size: 13px;
}
</string>
     </property>
     <item>
      <property name="text">
       <string>Image 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Image 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Image 3</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>60</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter Book Title</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bookTitle">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>601</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="bookAuthor">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>208</y>
       <width>601</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>245</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777185</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Enter Publisher Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bookPublisher">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>245</y>
       <width>601</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>208</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777185</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Enter Author Name</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>282</y>
       <width>601</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit {
	padding: 5px;
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>191</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="backIcon">
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: rgb(231, 138, 107);
}</string>
       </property>
       <property name="text">
        <string>go back</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="backBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border-radius:15px;
	font: 14px &quot;Montserrat&quot;;
}

QPushButton:hover {
	text-decoration: underline;
}</string>
       </property>
       <property name="text">
        <string>Go Back</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="logoImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/resources/images/bblLogo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="pageDesc">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>971</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter the details of the new book you want to add to the library catalogue.</string>
    </property>
   </widget>
   <widget class="QLabel" name="pageTitle">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size: 24px;
	font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Add New Book</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>bookTitle</tabstop>
  <tabstop>bookGenre</tabstop>
  <tabstop>bookCopies</tabstop>
  <tabstop>selectedImage_3</tabstop>
  <tabstop>bookAuthor</tabstop>
  <tabstop>bookPublisher</tabstop>
  <tabstop>confirmBookDetails</tabstop>
  <tabstop>backBtn</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
